py3-jupyterhub-idle-culler.yaml

```diff
@@ -0,0 +1,38 @@
+# Generated from https://pypi.org/project/jupyterhub-idle-culler/
+package:
+  name: py3-jupyterhub-idle-culler
+  version: 1.2.1
+  epoch: 0
+  copyright:
+    - license: 3-BSD
+  dependencies:
+    runtime:
+      - py3-tornado
+      - py3-python-dateutil
+      - python-3
+
+environment:
+  contents:
+    packages:
+      - ca-certificates-bundle
+      - wolfi-base
+      - busybox
+      - build-base
+      - python-3
+      - py3-setuptools
+
+pipeline:
+  - uses: fetch
+    with:
+      expected-sha256: d80822982d2590cb876d01849cee028943fdbdcc991c0f02fce45af3f40dd415
+      uri: https://files.pythonhosted.org/packages/e8/be/f225c5d4cd186f793e46ababecee6ee62b2a57906a0ec1fc0453cda6c6ed/jupyterhub-idle-culler-${{package.version}}.tar.gz
```

Comment:
Is there a reason we're not pulling this down from git?
https://github.com/jupyterhub/jupyterhub-idle-culler